_data-prepper/peer-forwarder.md

```diff
@@ -0,0 +1,158 @@
+---
+layout: default
+title: Peer Forwarder
+nav_order: 12
+---
+
+Peer Forwarder is an HTTP service which performs peer forwarding of `Event` between Data Prepper nodes for aggregation. Currently, supported by `aggregate`, `service_map_stateful`, `otel_trace_raw` processors.
+
+Peer Forwarder groups events based on the identification keys provided the processors. For `service_map_stateful` and `otel_trace_raw` it's `traceId` by default and can not be configured. It's configurable for `aggregate` processor using `identification_keys` configuration option. You can find more information about identification keys [here](https://github.com/opensearch-project/data-prepper/tree/main/data-prepper-plugins/aggregate-processor#identification_keys).
+
+---
+
+Right now, peer discovery is provided by either a static list or by a DNS record lookup or AWS Cloudmap.
+
+# Discovery modes
+
+## Static
```

Comment:
Let’s make sure to place a line after each heading and before each code block. I’ve seen this break page rendering